annotation-check/src/main.ts

```diff
@@ -36,7 +36,21 @@ async function run(): Promise<void> {
     if (check_run.output.title) {
       core.info(`- ${check_run.output.title}: ${check_run.output.summary}`)
     }
-    count += check_run.output.annotations_count
+    if (check_run.output.annotations_count > 0) {
+      const annotations = await octokit.rest.checks.listAnnotations({
+        owner,
+        repo,
+        check_run_id: check_run.id
+      })
+
+      for (const annotation of annotations.data) {
+        core.info(`${annotation.path}:${annotation.start_line} - ${annotation.message}`)
+	if (!annotation.message.startsWith("ubuntu-latest")) {
```

Comment:
Same happens from time to time for the other images. So maybe look for the other parts that are more generic.

Reply:
I'm not entirely convinced GH is ever going to use this format again, and we only use windows-latest & ubuntu-latest as far as I can see (and I don't think windows-latest is going to change in a meaningful way anytime soon). But I've updated it to be generic anyway. Unsure if better.